### PR TITLE
eslint no-extraneous-class: improve mapperTabPage

### PR DESCRIPTION
**Pull request by thiagowfx**
Bug: #392
Docs: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-extraneous-class.md

### CODE REVIEW

File: src/bidiTab/mapperTabPage.ts [hunk 1]
```diff
@@ -56,21 +56,20 @@ function findOrCreateTypeLogContainer(logType: string): HTMLElement {
   return document.getElementById(containerId)!;
 }
 
-export class MapperTabPage {
-  static generatePage() {
-    // If run not in browser (e.g. unit test), do nothing.
-    if (!globalThis.document?.documentElement) {
-      return;
-    }
-    window.MapperTabPage = MapperTabPage;
```
Comment (thiagowfx) [inline src/bidiTab/mapperTabPage.ts:65]: I did not see any harm in removing this assignment.

Comment (OrKoN) [reply src/bidiTab/mapperTabPage.ts:65]: I feel like it must have been needed for smth, let's ask Vladimir :)

Comment (thiagowfx) [reply src/bidiTab/mapperTabPage.ts:65]: cc @nechaev-chromium

Comment (nechaev-chromium) [reply src/bidiTab/mapperTabPage.ts:65]: window.MapperTabPage is not used by ChromeDriver.

**Review by nechaev-chromium — APPROVED** [review]

File: src/bidiTab/mapperTabPage.ts (hunk 1 above)
Comment (thiagowfx) [reply src/bidiTab/mapperTabPage.ts:65]: Turns out this was needed. It caused the following issue: https://github.com/GoogleChromeLabs/chromium-bidi/issues/433. I am readding it back on https://github.com/GoogleChromeLabs/chromium-bidi/pull/434

**Pull request merged**